### DISCUSSION TITLE
Add Menuitems on DWIN

### DISCUSSION THREAD

**Discussion by Dirk-76** [mriscoc/Ender3V2S1#1349] [Q&A]
Hello,

is it possible to add a Menuitem on a DWIN for me?

Marlin Custom menu doesn't work for DWIN.

Need a "Turn OFF Printer" that sends M81, for a skr Mini e3 v3 with a btt relays 1.2.

It only has a reset relay function per button that the printer turns on.
Off only per g-code.

Too add it as quick menu item where realy fine.

In which files I have to take a look?

Need the Dwin FW  a change too then to add this? (Hope and think not ... or?)

till I have work me in that, I can help me out with a g-code M81, that I select in prints...

But per menu where a better solution,  that my inner Monk shut up. 

Sorry for this.

VaraEdo

**Comment by mriscoc** [comment]
You could write something similar to a [configuration file](https://github.com/mriscoc/Ender3V2S1/wiki/Configuration-files) and named it Turn_Off.gco. Put your custom script/gcode inside that file, then you can select that file to execute the script from the display.

To be able to use M81 you need to compile a custom version with PSU_CONTROL enabled and configure the required PS_ON_PIN
